geomstats/information_geometry/normal.py

```diff
@@ -11,17 +11,19 @@
     PoincareHalfSpace,
     PoincareHalfSpaceMetric,
 )
+from geomstats.information_geometry.information_manifold import InformationManifold
 
 
-class NormalDistributions(PoincareHalfSpace):
+class NormalDistributions(PoincareHalfSpace, InformationManifold):
     """Class for the manifold of univariate normal distributions.
 
     This is upper half-plane.
     """
 
     def __init__(self):
         super().__init__(dim=2)
-        self.metric = FisherRaoMetric()
+        self.shape = (2,)
+        self.metric = NormalMetric()
```

Comment:
I think we should change `PoincareHalfSpace`  to accept metric as input as pass this in `super`.

Reply:
PoincareHalfSpace already has its own metric, which the PoincareHalfSpaceMetric from which NormalMetric inherits.
I think we can leave it this way to clarity in the inheritances